### PR TITLE
zh-trans: add setup-tools/kubeadm/kubeadm-reset.md&kubeadm-version.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add /zh/docs/reference/setup-tools/kubeadm/kubeadm-reset.md

### CODE REVIEW

**Comment by xichengliudui** [comment]
Job log:

*Building sites … ERROR 2018/12/16 03:05:59 [zh] no Resource or Page matching "generated/kubeadm_reset.md*".
ERROR 2018/12/16 03:06:00 [zh] no Resource or Page matching "generated/kubeadm_version.md*".*

**Comment by tengqm** [comment]
kubeadm-version.md 的路径不对。

**Comment by AdamDang** [comment]
@tengqm 
检查了一下，generated/kubeadm_version.md  是存在的
generated/kubeadm_reset.md 确实不存在

**Comment by tengqm** [comment]
@AdamDang This PR is adding a new kubeadm-version.md file, not sure if that is correct. But, anyway, that is the reason of CI failure. Please help fix it.

**Comment by AdamDang** [comment]
@tengqm I've solved the ci error.
One generated files are not there. So I translated them. :) It's OK now!

**Comment by AdamDang** [comment]
I've solved the ci error. root cause was that  generated file 'kubeadm_reset.md' was not there.

File: content/zh/docs/reference/setup-tools/kubeadm/kubeadm-version.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+<!--
+This command prints the version of kubeadm.
+-->
+
+此命令用来查询 kubeadm 的版本。
+
+{{% /capture %}}
+
+{{% capture body %}}
+{{< include "generated/kubeadm_version.md" >}}
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/kubeadm-version.md]: This line requires the "generated/kubeadm_version.md" file, which is not found yet.

Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/kubeadm-version.md]: just now I added "generated/kubeadm_version.md", but this caused conflicts. So I deleted it and  ci passed.

**Comment by tengqm** [comment]
@AdamDang I think you may need a rebase of the PR to release-1.12 again. The kubeadm_version.md file was only added yesterday.

**Comment by tengqm** [comment]
@AdamDang I have just pulled your PR into my repo and tried a manual rebase onto release-1.12. During rebase, I found the kubeadm_version.md file added, deleted, added again. That might be the reason behind the dilemma we have here. Created new PR for testing: https://github.com/kubernetes/website/pull/11796.

Let's see if we can fix the chaos.

**Comment by AdamDang** [comment]
It's OK now. :)

**Comment by AdamDang** [comment]
At beginning, the referenced files were missed. 
I added them in this PR and found those files were added by other one, so conflicts happened.  
Just now I did a manual rebase, Solved!

**Comment by tengqm** [comment]
Correct. You had to do a manual rebase. That is the reason I hate batch adding files aimlessly. The experience we learned from this is that we may need to do a manual rebase/squash everytime when we commit an updated version of the PR. Avoid committing separate commits in the same PR when possible, because it will make your life very difficult.

**Comment by AdamDang** [comment]
" it will make your life very difficult"
Can't agree more!
:)

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+
+<table style="width: 100%; table-layout: fixed;">
+  <colgroup>
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--cert-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/pki"</td>
```
Comment (Mr-Linus) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md]:
```suggestion
       <td colspan="2">--cert-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: "/etc/kubernetes/pki"</td>
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">到存储证书的目录的路径。如果指定了，需要清除此目录。</td>
+    </tr>
+    
+    <tr>
+      <td colspan="2">--cri-socket string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/var/run/dockershim.sock"</td>
```
Comment (Mr-Linus) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md]:
```suggestion
       <td colspan="2">--cri-socket string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;默认值: "/var/run/dockershim.sock"</td>
```

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">在不提示确认的情况下重置节点。</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">help for reset</td>
```
Comment (Mr-Linus) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md]: 这个可以翻译下

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md [hunk 5]
```diff
@@ -102,7 +102,7 @@ kubeadm reset [flags]
       <td colspan="2">-h, --help</td>
     </tr>
     <tr>
-      <td></td><td style="line-height: 130%; word-wrap: break-word;">help for reset</td>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">重置帮助</td>
```
Comment (tengqm) [inline content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_reset.md]: reset 操作的帮助信息

**Pull request merged**